be/src/util/compression/compression_utils.h

```diff
@@ -36,6 +36,26 @@ class CompressionUtils {
         }
         return CompressionTypePB::UNKNOWN_COMPRESSION;
     }
+
+    static CompressionTypePB to_compression_pb(const std::string& ext) {
```

Comment:
why not use a map<string, compressionTypePB> to store such info?

Reply:
Because number of enumeration is small, so I choose simple way to handle it.